deploy/mobileclient_admin_cluster_role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: mobileclient-admin-cluster
```

Comment:
I don't think this is a very accurate name: it's not related to the other "mobileclient-admin" Role[Binding] pair, and it doesn't really give the ability to "admin" mobileclients.

How about something like "mobileclient-users" or something?